docs/_docs/installation.md

```diff
@@ -4,42 +4,51 @@ description: Official guide to install Jekyll on macOS, GNU/Linux or Windows.
 permalink: /docs/installation/
 ---
 
-Installing Jekyll should be straight-forward if all requirements are met.
-
+- [Requirements](#requirements)
 - [Install on macOS](#macOS)
-- [Install on GNU/Linux](#Linux)
 - [Install on Windows](../windows/)
+- [Upgrade Jekyll](#upgrade-jekyll)
+
+Installing Jekyll should be straight-forward if all requirements are met.
+
+## Requirements
+
+Before you start, make sure your system has the following:
+
+- [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.2.5 or above, including all development headers (ruby installation can be checked by running `ruby -v`)
+- [RubyGems](https://rubygems.org/pages/download) (which you can check by running `gem -v`)
+- [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn't have them installed, which you can check by running `gcc -v`,`g++ -v`  and `make -v` in your system's command line interface)
 
 ## Install on macOS {#macOS}
 
-We only cover latest macOS High Sierra 10.13 here, who ships a new Ruby version, older systems will need to install Homebrew, see below.
+We only cover latest macOS High Sierra 10.13 here, who comes with Ruby 2.3.3, older systems will need to [install a more recent Ruby version via Homebrew](#homebrew).
 
 First, you need to [install Xcode from the AppStore](https://itunes.apple.com/fr/app/xcode/id497799835?mt=12). Then to install the command-line tools, open a terminal and run:
 
 ```sh
 xcode-select --install
 ```
 
-Check the pre-installed ruby and [RubyGems](https://rubygems.org/pages/download) versions:
+Check your Ruby version, and upgrade [RubyGems](https://rubygems.org/pages/download):
 
 ```sh
 ruby -v
 2.3.3
-gem --version
-2.7.6
+
+gem update --system
 ```
 
-Great, let's install Jekyll, we'll also need bundler to help us handle plugins and themes:
```

Comment:
Feels like this should be two sentences.

"Great, let's install Jekyll. We'll also need Bundler to help us handle plugins and themes:"

From that point on, continue.